optaplanner-core/src/main/java/org/optaplanner/core/config/localsearch/decider/acceptor/LocalSearchAcceptorConfig.java

```diff
@@ -380,4 +381,9 @@ public LocalSearchAcceptorConfig copyConfig() {
         return new LocalSearchAcceptorConfig().inherit(this);
     }
 
+    @Override
+    public void visitReferencedClasses(Consumer<Class<?>> classVisitor) {
+        // No referenced classes
+    }
```

Comment:
Maybe we provide a default empty impl to replace the abstract method? Only people who need to override would override, removing the need for these empty overrides.

But thinking about it again, maybe this way is better. Author of each new config will be forced to think about implementing this method, and that's probably a good thing.

Reply:
That the idea (and also why I didn't create a default version in PhaseConfig; I want to force subclasses to implement these methods so writers known they must add the classes to reduce probability of bugs).